base/workerpool.jl

```diff
@@ -232,12 +232,14 @@ serialize(s::AbstractSerializer, cp::CachingPool) = throw(ErrorException("Cachin
 """
     CachingPool(workers::Vector{Int})
 
-An implementation of an `AbstractWorkerPool`. `remote`, `remotecall_fetch`, `pmap` and other
-remote calls which execute functions remotely, benefit from caching the serialized/deserialized
-functions on the worker nodes, especially for closures which capture large amounts of data.
+An implementation of an `AbstractWorkerPool`.
+[`remote`](:func:`remote`), [`remotecall_fetch`](:func:`remotecall_fetch`),
+[`pmap`](:func:`pmap`) (and other remote calls which execute functions remotely)
+benefit from caching the serialized/deserialized functions on the worker nodes,
+especially closures (which capture large amounts of data).
```

Comment:
the benefit of caching is most significant when the amount of captured data is large, but that's not necessarily very common